cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -193,31 +213,94 @@ func waitForCreateJob(client BackplaneApi.ClientInterface, job *BackplaneApi.Job
 	pollErr := wait.PollImmediate(10*time.Second, time.Duration(600)*time.Second, func() (bool, error) {
 		fmt.Printf(".")
 
-		// ========= Get the current job ============
-		jobResp, err := client.GetRun(context.TODO(), options.clusterId, *job.JobId)
-
+		// Get the current job
+		status, err := getJobStatus(client, job)
 		if err != nil {
 			return false, err
 		}
 
-		formatJobResp, err := BackplaneApi.ParseGetRunResponse(jobResp)
+		// Check if the job is in the expected status
+		if status == BackplaneApi.JobStatusStatusSucceeded {
+			statusMessage = "Job Succeeded"
+			return true, nil
+		}
+		if status == BackplaneApi.JobStatusStatusFailed {
+			statusMessage = "Job Failed"
+			return true, nil
+		}
+
+		return false, nil
+	})
+
+	return statusMessage, pollErr
+}
+
+// fetchJobLogs stream the log of the job to the console output when the job status is Running, Succeeded or Failed
+func fetchJobLogs(client BackplaneApi.ClientInterface, job *BackplaneApi.Job) error {
 
+	pollErr := wait.PollImmediate(5*time.Second, 1*time.Minute, func() (bool, error) {
+		fmt.Printf(".")
+
+		// Get the current job
+		status, err := getJobStatus(client, job)
 		if err != nil {
 			return false, err
 		}
 
-		if *formatJobResp.JSON200.JobStatus.Status == BackplaneApi.JobStatusStatusSucceeded {
-			statusMessage = "Job Succeeded"
+		// Check if the job is in the expected status
+		switch status {
+		case BackplaneApi.JobStatusStatusPending:
+			return false, nil
+		case BackplaneApi.JobStatusStatusRunning:
+			fmt.Println("")
 			return true, nil
-		}
-
-		if *formatJobResp.JSON200.JobStatus.Status == BackplaneApi.JobStatusStatusFailed {
-			statusMessage = "Job Failed"
+		case BackplaneApi.JobStatusStatusSucceeded:
+			fmt.Println("")
+			return true, nil
+		case BackplaneApi.JobStatusStatusFailed:
+			fmt.Println("")
 			return true, nil
+		default:
+			return false, fmt.Errorf("job is not ready with logs")
 		}
 
-		return false, err
 	})
 
-	return statusMessage, pollErr
+	if pollErr != nil {
+		return pollErr
+	}
+
+	version := "v2"
+	follow := true
+
+	resp, err := client.GetJobLogs(context.TODO(), options.clusterId, *job.JobId, &BackplaneApi.GetJobLogsParams{Version: &version, Follow: &follow})
```

Comment:
In this case, logs are printed to the console when the job is finished. 
ideally, you can stream logs here 
```
case BackplaneApi.JobStatusStatusPending:
//Get the intermediate logs 
// append diff of the logs 
			return false, nil
```

Reply:
All good .. I tested the feature and logs show when the job status is `Running` |`Succeeded` | `Completed`